README.md

```diff
@@ -1,18 +1,20 @@
-The source code for the Pomona Shuttle Share application. Created by Jesse Pollak, maintained by ASPC.
```

Comment:
Don't remove the created by Jesse Pollak line. IMO he deserves that credit even in the README.